src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/replication/ReplicationFailedAction.java

```diff
@@ -32,14 +37,14 @@
  */
 public class ReplicationFailedAction extends InvisibleAction {
 
-    private String reason;
+    private BlockedItem blockedItem;
```

Comment:
You should probably make BlockedItem serializable just in case, if nothing else just to mark that it will/could be serialized at some point.